src/components/Card.jsx

```diff
@@ -0,0 +1,22 @@
+const Card = ({ review }) => {
+    const { review_id, title, votes, comment_count, review_img_url, created_at, review_body, owner, designer, category } = review;
+
+    const reviewBody = review_body.slice(0, 100) + ' . . .'
+    
+    return <div className="card" id="review_id">
+        <h3 className="card-title">{ `${title}` }</h3>
+        <img src={`${review_img_url}`} alt="{title}" />
+        <button className='votes'>Votes: {votes} </button>
+        <button className="comments">Comments: {comment_count}</button>
+        <div className="category-chip"> {category} </div>
+        <div className="card-info">
+            <div className="card-info-top">
+                <p className="card-reviewer"> {`${owner}`} </p>
+                <p className="card-date">{`${created_at}`}</p>
+            </div>
+            <p className="card-review-body"> {`${reviewBody}`} </p>
+        </div>
+    </div>
+}
+
```

Comment:
I like your review card styling! I would maybe add a bit of padding so the letters don't go right up to the edge of the element they're in